packages/gatsby-theme-docs/src/components/notifications.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import UnstyledInformationIcon from '../icons/information-icon.svg';
+import UnstyledWarningIcon from '../icons/warning.svg';
+import UnstyledErrorIcon from '../icons/error.svg';
+import createStyledIcon from '../utils/create-styled-icon';
+import markdown2React from '../utils/markdown-2-react';
+import { pxToRem, colors, tokens, dimensions } from '../design-system';
+
+const Container = styled.div`
+  display: flex;
+  width: 100%;
+  padding: ${dimensions.spacings.m};
+  margin: ${dimensions.spacings.s} 0;
+  border: ${pxToRem('1px')} solid ${colors.light.borderInfo};
```

Comment:
The border should just stay to 1px, at least that's what we do at the moment. If we were to change that, we should do that in a separate PR. Also the value should be defined in the design-system, so that we don't need to use `pxToRem` directly.

```suggestion
  border: 1px solid ${colors.light.borderInfo};
```